client/client.go

```diff
@@ -97,6 +97,7 @@ import (
 	"regexp"
 	"time"
 
+	"github.com/cloudflare/cfssl/log"
```

Comment:
we should remove this import and all other logs in this file should be using `logrus`